tests/queries/0_stateless/02417_opentelemetry_insert_on_distributed_table.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest, distributed
+
+set -ue
+
+unset CLICKHOUSE_LOG_COMMENT
```

Comment:
Looks like this is not required? (maybe the test was relying on `log_comment` before, but not now)